miio/viomivacuum.py

```diff
@@ -439,3 +613,29 @@ def carpet_mode(self, mode: ViomiCarpetTurbo):
     def fan_speed_presets(self) -> Dict[str, int]:
         """Return dictionary containing supported fanspeeds."""
         return {x.name: x.value for x in list(ViomiVacuumSpeed)}
+
+    @command(click.argument("state", type=bool))
+    def set_repeat(self, state: bool):
+        """Set or Unset repeat mode."""
+        return self.send("set_repeat", [int(state)])
+
+    @command()
+    def get_maps(self) -> List[Dict[str, Any]]:
+        """Return map list."""
```

Comment:
Could you add an example response for this request? If the obtained information could be useful for some other features (e.g., "use this map", or something similar), it may make sense to create a container class out of this that can be re-used.

Reply:
done